components/datetime/tests/data/gen.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+cargo run -p icu_datagen -- \
+--keys "datetime/gregory/datelengths@1" "datetime/gregory/datesymbols@1" "datetime/timelengths@1" "datetime/timesymbols@1" "decimal/symbols@1" "time_zone/formats@1" "time_zone/specific_short@1" \
```

Comment:
Question: Was this code being run in CI before, and it is being run in CI now?

Reply:
it was not

Reply:
and is not